ExampleFhirClient/AccessTokenHelper.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IdentityModel.Tokens;
+using System.Net.Http;
+using System.Security.Claims;
+using System.Security.Cryptography.X509Certificates;
+
+namespace ExampleFhirClient
+{
+	static class AccessTokenHelper
+	{
+		public static string GetAccessToken( Uri baseAddress )
+		{
+			var cert = GetClientCertificate( "CN=jwt.careevolution.com" );
+			var clientId = "JWTClientCredentials";
+			var tokenvalue = CreateClientCredentialsJWT( cert, clientId, baseAddress );
+
+			using ( var client = new HttpClient { BaseAddress = baseAddress } )
+			{
+				var grantResponse = RequestClientCredentialsJWT( client, clientId, tokenvalue );
+
+				var responseData = grantResponse.Content.ReadAsAsync<Dictionary<string, object>>().Result;
+				if ( !responseData.TryGetValue( "access_token", out var accessToken ) )
+				{
+					return null;
+				}
+				return accessToken as string;
+			}
+		}
+
+		private static X509Certificate2 GetClientCertificate( string certName )
+		{
+			var store = new CertificateStore();
+			var cert = store.GetMostRecentDistinguishedNameCertificate( StoreName.My, StoreLocation.LocalMachine, certName );
+			return cert;
+		}
+
+		private static string CreateClientCredentialsJWT( X509Certificate2 signingCertificate, string clientId, Uri baseAddress, string audience = null, string[] omitHeaders = null )
```

Comment:
`audience` and `omitHeaders` are always null - so maybe remove them? Again, so that the example is simpler?

Reply:
removed and simplified certificate access code.